name.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "Nareauphol (Gavin) Liu"
+}
```

Comment:
Please have this file under the `data` folder, and name the file to be `your_name.json` not literal `name.json`.